docs/05-module-block-5/02-learning-goals.adoc

```diff
@@ -14,54 +14,34 @@ Zudem vermittelt er Strategien, um mehrere Modelle, und die daraus erwachsenden
 
 [[LZ-5-2]]
 ==== LZ 5-2: Vor- und Nachteile eines Team-übergreifenden Modells abwägen können
-* Die TN kennen Conway's Law footnote:7[Cf.: <<conway>>] und können es auf die (Kommunikations-)Struktur von Entwicklungs-Teams und Software-Artefakten anwenden.
+* Die TN kennen Conway's Law footnote:7[Cf.: <<conway>>] und seine Auswirkungen auf Software-Architektur.
 * Die TN verstehen, dass von mehreren Teams genutzte Modelle teamübergreifende Kommunikation und Koordination (z. B. beim Deployment) erfordern.
 * Die TN verstehen, dass Modelle aufgeteilt werden müssen, damit mehrere Entwicklungsteams unabhängig voneinander arbeiten können.
 * Die TN verstehen, dass definierte Modellgrenzen eine wichtige Rolle dabei spielen, den Kommunikationsaufwand gering zu halten.
 * Die TN wissen, dass Begriffe der Fachsprache in verschiedenen Kontexten verschiedene Bedeutungen haben können.
 * Die TN wissen, dass lokale Modellkonsistenz unabhängig von anderen Modellen aufrechterhalten werden kann.
 
 [[LZ-5-3]]
-==== LZ 5-3: Modellgrenzen von Bounded Contexts in einer Context Map beschreiben können
+==== LZ 5-3: Vom Problem- zum Lösungsraum kommen können
+* Die TN kennen die Begriffe "Problem-" und "Lösungsraum" und den Unterschied.
```

Comment:
Ich fände es besser, hier tatsächlich "Problemraum" stehen zu haben statt "Problem-", um es expliziter zu machen. Dadurch, dass die Wörter in Anführungszeichen stehen, wirkt das sonst etwas verwirrend.